### ISSUE TITLE
Vertical participant hit box should contain the top bar

### ISSUE THREAD

**Issue by barmac** [bpmn-io/bpmn-js#2093]
### Describe the Bug

The hit box created for a vertical participant is the same as for a horizontal one which leads to confusion.

### Steps to Reproduce

https://github.com/bpmn-io/bpmn-js/assets/28307541/d8277fa8-0373-4e86-966a-2d1eed45876f

### Expected Behavior

The hit box should contain the top bar and not the space on the left side.

### Environment

 - Browser: [e.g. IE 11, Chrome 69]
 - OS: [e.g. Windows 7]
 - Library version: [e.g. 2.0.0]

**Comment by sombrek** [comment]
This is easy to fix, just a small change in [BpmnInteractionEvents](https://github.com/bpmn-io/bpmn-js/blob/develop/lib/features/interaction-events/BpmnInteractionEvents.js).

However, I'm not exactly sure how to tests this. Existing tests don't seem to cover the dimensions or clicks on the hit zone.

**Comment by barmac** [comment]
Indeed, it looks like a small change. I think the proper tests should also check the dimensions of the hit zones. If you add them for both horizontal and vertical lanes and pools, that will be great.

**Comment by sombrek** [comment]
@barmac Thanks for your feedback.

Could you please give me a hint on how I can get the dimensions of the hit zone?
Based on `BpmnInteractionEventsSpec` I tried to get the hit zone element via `domQueryAll('.' + HIT_ALL_CLS, gfx)`, where `gfx = elementRegistry.getGraphics(lane);`. This does seem to work, but the result is just an empty object `{"0":{}}`, so I can't check its properties.

**Comment by barmac** [comment]
Feel free to use this branch for base of your contribution: https://github.com/bpmn-io/bpmn-js/tree/2093-hitboxes

**Comment by barmac** [comment]
Closed via https://github.com/bpmn-io/bpmn-js/pull/2105
